src/react-components/info-dialog.js

```diff
@@ -217,6 +218,26 @@ class InfoDialog extends Component {
           </div>
         );
         break;
+      case InfoDialog.dialogTypes.webvr_recommend:
+        dialogTitle = "Enter in VR";
+        dialogBody = (
+          <span>
+            To enter Hubs with Oculus or SteamVR, you can use Firefox.
+            <p />
+            <a className="info-dialog--action-button" href="https://www.mozilla.org/firefox">
```

Comment:
should this be `target="_blank" rel="…"`?

Reply:
haha I was wondering if you'd notice this -- I decided to not do that specifically here because my expectation is that you are going to bounce off of hubs to download FF at this point, and restart your session there. However the other links open in a new tab b/c they are not primary CTA